src/PDE/CompFlowProblem/UserDefined.h

```diff
@@ -0,0 +1,187 @@
+// *****************************************************************************
+/*!
+  \file      src/PDE/CompFlowProblem/UserDefined.h
+  \author    J. Bakosi
```

Comment:
Aren't we removing these to support a more traditional open source development model?